src/cloud/project/new-relic.md

```diff
@@ -175,7 +208,42 @@ All collected data details the time spent on an action transmitting data, querie
 {:.bs-callout-info}
 You can learn more about using the New Relic APM and Infrastructure agents to collect and analyze application data from the [New Relic APM] and [New Relic Infrastructure][New Relic infrastructure] documentation.
 
-## Monitor performance with alert policies
+### View and analyze log data
+
+After you log in to your New Relic account, you can use the New Relic Logs UI to search across the aggregated log data to troubleshoot application, infrastructure, CDN, and WAF errors. Also, you can connect the log data with other data collected by New Relic APM and Infrastructure services to create charts, dashboards, and alerts to manage Magento application and cloud service operations.
```

Comment:
It starts with "After you log in", but in the flow of this topic, you are in the Use/Using part already. Just an observation on flow. Also, New Relic calls Logs an app.

```suggestion
You can use the New Relic Logs application to search across the aggregated log data and troubleshoot application, infrastructure, CDN, and WAF errors. Also, you can connect the log data with other data collected by New Relic APM and Infrastructure services to create charts, dashboards, and alerts to manage Magento application and cloud service operations.
```